app/poros/nytimes_results.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../services/nytimes_service.rb'
+require_relative './obituary.rb'
+
+class NytimesResults
+  def get_obituaries
+    json = NytimesService.new.get_obituaries
+    json[:response][:docs].map do |obituary_data|
+      Obituary.new(format_obituary_data(obituary_data))
+    end
+  end
+
+  private
+  def format_obituary_data(obituary_data)
+    {
+      headline: obituary_data[:headline][:main],
+      abstract: obituary_data[:abstract],
+      web_url: obituary_data[:web_url],
+      pub_date: obituary_data[:pub_date],
+      word_count: obituary_data[:word_count]
+    }
```

Comment:
I tried to grab the data that we might find most useful, we will need to explore the API more to see if we can get anything better.

Reply:
For now I think this is great!

Reply:
Per our phone convo earlier, it looks like we will have to return different information based on the search. More specifically, for covid we will need [:snippet] instead of [:lead_paragraph]. Perhaps we just grab both so that the rails app has everything and then parse based on the needs of that particular query?

Reply:
Cool sounds good. I'll restructure the conn/request before we merge so that It is more flexible and we can add all of the searches we want.

Reply:
Looks good Jack! Nice work here.